src/services/textChanges.ts

```diff
@@ -73,8 +88,8 @@ namespace ts.textChanges {
     export interface ConfigurableStartEnd extends ConfigurableStart, ConfigurableEnd {}
 
     export const useNonAdjustedPositions: ConfigurableStartEnd = {

```

Comment:
How much work would it be to stop referring to "adjusted" positions outside of this file?  It looks like you've done most of it already.